src/components/SubmissionForm/SubmissionForm.scss

```diff
@@ -17,20 +17,22 @@ form {
   padding: 30px;
   border-radius: 10px;
   box-shadow: 10px 5px 5px rgb(46, 70, 70);
-}
 
-label {
+  label {
   color: #FFF;
   margin-top: 1vh;
   margin-bottom: .5vh;
 }
 
-h2 {
-  font-size: 3vh;
-  color: #FFF;
-  text-align: center;
+  h2 {
+    font-size: 3vh;
+    color: #FFF;
+    text-align: center;
+  }
```

Comment:
Neat! Thank you for making this switch and teaching me a new CSS trick! I definitely should have targeted the element by className. Thank you for catching that.